### PR TITLE
Add icons from designs to CategoryIcon and PinIcon

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

**Comment by ahie** [comment]
Reopening PR to restart the pipeline..

File: src/common/ui-components/Map/PinIcon/PinIcon.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.offset {
```
Comment (marharyta) [inline src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: any specific reason for this offset?

Comment (ahie) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: It's to align the point of the pin with the coordinates of the feature.

Comment (marharyta) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: why exactly this much?

Comment (marharyta) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: Can I see the image?

Comment (ahie) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: Its height and width / 2 of the pins.

Comment (ahie) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: https://helsinkicity.slack.com/archives/CMKU6RAUX/p1584952087065700

Comment (marharyta) [reply src/common/ui-components/Map/PinIcon/PinIcon.module.scss:1]: Thanks! It was just hard for me to see the difference from what it used to be before

**Pull request merged**